app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -108,10 +114,17 @@ invitationForm = React.createClass
         @errorFor fieldName
 
   errorFor: (fieldName) ->
-    @props.fields[fieldName].error
+    if fieldName == 'domainIds'
```

Comment:
Idd beetje jammer, maar zoals in de comment hieronder staat komt dit door een rariteit van redux-form versie 5 en kan dit hopelijk netter met de nieuwste versie (is al ticket voor)

Door null door te geven krijg je weer een extra branch in `renderErrorFor` waarmee je het probleem volgens mij alleen maar doorschuift.